frontend/pages/statistics.tsx

```diff
@@ -64,7 +65,7 @@ const Statistics = ({ stats }: { stats: Statistics }): JSX.Element => {
                 content: {
                   type: "text",
                   text: stats.downloads?.toLocaleString(
-                    i18n.language.substring(0, 2),
+                    getIntlLocale(i18n.language),
```

Comment:
but I don't think this is going to work. The web platfrom uses the hyphenated form, `en-GB`. But this function appears to return the underscored form.

Reply:
Oh I see, this is actually returning a `Locale` object.